end

```diff
@@ -0,0 +1,530 @@
+[33mcommit 6bb102f0bbf6c41ac1b766d8ee1c08e615262859[m[33m ([m[1;36mHEAD -> [m[1;32msum[m[33m, [m[1;31morigin/sum[m[33m)[m
```

Comment:
It would be better, if this thing(file) is removed.

Reply:
Yeah! I've removed it.